staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -1394,6 +1394,11 @@ func (c *Cloud) Instances() (cloudprovider.Instances, bool) {
 	return c, true
 }
 
+// InstancesV2 returns an implementation of InstancesV2 for Amazon Web Services.
+func (c *Cloud) InstancesV2() (cloudprovider.InstancesV2, bool) {
+	return c, true
```

Comment:
set this to false and let aws OWNERs flip this on?

Reply:
All set as false, it's a method with low risk.